### PR TITLE
Added stroke support.

### PR DESCRIPTION
**Pull request by mmatyas**
You can now draw the whiskers of
the tiger.

### CODE REVIEW

File: src/azure.rs [hunk 1]
```diff
@@ -478,6 +478,8 @@ pub fn AzDrawTargetStrokeLine(aDrawTarget: AzDrawTargetRef, aStart: *mut AzPoint
 
 pub fn AzDrawTargetFill(aDrawTarget: AzDrawTargetRef, aPath: AzPathRef, aPattern: AzPatternRef, aOptions: *mut AzDrawOptions);
 
+pub fn AzDrawTargetStroke(aDrawTarget: AzDrawTargetRef, aPath: AzPathRef, aPattern: AzPatternRef, aStrokeOptions: *mut AzStrokeOptions, aDrawOptions: *mut AzDrawOptions);
```
Comment (jdm) [inline src/azure.rs]: These should be *const, not *mut.

**Comment by mmatyas** [comment]
Something like this?

**Comment by jdm** [comment]
Yep! Squash and I'll merge.

**Comment by mmatyas** [comment]
Ok, done.

**Pull request merged**